java/client/src/main/java/glide/api/commands/StringBaseCommands.java

```diff
@@ -34,6 +34,26 @@ public interface StringBaseCommands {
      */
     CompletableFuture<String> get(String key);
 
+    /**
+     * Gets a string value associated with the given <code>key</code> and deletes the key. {@see
+     * #get}, except for the fact that it also deletes the key on success (if and only if the value of
```

Comment:
Not sure that we need the second phrase. @aaron-congo, what do you think?

Reply:
I think we can remove it